docs/runbooks/KubeMacPoolDuplicateMacsFound.md

```diff
@@ -0,0 +1,32 @@
+# KubeMacPoolDuplicateMacsFound
+
+## Meaning
+
+KubeMacPool is responsible to allocate MAC Addresses and prevent MAC Address conflicts.
+Upon starting KubeMacPool, it scans the cluster for MAC Addresses in VMs set on the managed namespaces.
+In case it finds duplicates it will raise this alert.
+
+## Impact
+
+Having a duplicate MAC Addresses can break the requirement that each interface on the same LAN,
+should have a unique MAC Address.
+
+## Diagnosis
+
+- Find the `kubemacpool-mac-controller` namespace and pod,
+  `kubectl get pod -A -l control-plane=mac-controller-manager --no-headers -o custom-columns=":metadata.namespace,:metadata.name"`
```

Comment:
This line appears in the same line when looking in the `Display rich format` option, I think you need to add a new line. also in the rest